components/brave_vpn/browser/connection/win/brave_vpn_helper/brave_vpn_helper_state.cc

```diff
@@ -76,4 +81,20 @@ std::wstring GetVpnServiceDisplayName() {
   return install_static::GetBaseAppName() + kBraveVpnServiceDisplayName;
 }
 
+bool IsNetworkFiltersInstalled() {
```

Comment:
How about adding below check or return false when both are not true at the start?
I think filter installation state has only valid when service is installed and running properly.
```
DCHECK(IsBraveVPNHelpServiceRunning());
DCHECK(IsBraveVPNHelperServiceInstalled());
```

Reply:
Hmm, it seems filter could not be installed while service is running during the `ScheduleExit()`?

Reply:
added dchecks